docs/website/docs/plus/features/transformations/executing.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Execute transformations
+description: Define and execute local transformations
+---
+
+dlt+ provides a powerful mechanism for executing transformations on your data using a locally spun-up cache. It automatically creates and manages the cache before execution and cleans it up afterward.
+
+A dlt+ Transformation consists of functions that modify data stored in a dlt+ Cache. These transformations can be implemented using:
```

Comment:
do we need to capitalize transformation in this case? or the cache for that matter? Because they're not product names, just features.